Arduino_Code/Arduino_Code.ino

```diff
@@ -93,18 +98,16 @@ void loop() {
     while (Serial.available())
       Serial.read();
     // Start time measurement
-    unsigned long startTime = micros();
+    uint16_t startTime = micros();
```

Comment:
micros() result wont fit into uint16_t, an uint32_t is needed here. Otherwise wrong hashrate will be reported.